src/types/markdown.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+var m      = require("mithril"),
+    editor = require("codemirror"),
+    Remarkable = require("remarkable"),
+    md = new Remarkable(),
```

Comment:
Since this is a local declaration and not an import it should be the last thing in the var statements.